WeatherWebService/WeatherWebService/Controllers/CityDetailsViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  CityDetailsViewController.swift
+//  WeatherWebService
+//
+//  Created by komal lunkad on 29/09/16.
+//  Copyright © 2016 komal lunkad. All rights reserved.
+//
+
+
+import UIKit
+
+class CityDetailsViewController: UIViewController {
+    @IBOutlet var textFieldCityName: UITextField!
+    var cityName:String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        textFieldCityName.becomeFirstResponder()
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func buttonCheckWeatherTapped(sender: AnyObject) {
+        cityName = textFieldCityName.text
+        if !isWhitespace(cityName!) {
+            self.showAlertView(Constants.AlertViewMessages.alertViewTitle,alertMessage: Constants.AlertViewMessages.whiteSpaceMessage)
+        }
+        else if !isAlphabetic(cityName!) {
+            self.showAlertView(Constants.AlertViewMessages.alertViewTitle,alertMessage: Constants.AlertViewMessages.onlyAlphabetMessage)
+        }
+        else{
+            pushViewController()
+        }
+    }
+    
+    //MARK:IsWhitespace
+    /** Checks whether the input is empty or not
+     * param name: input string to be checked
```

Comment:
- parameter , this is the required format.This creates the doc in the right format.Similar changes on other methods as well